modules/op-behavior-of-approvaltask-with-groups-and-users.adoc

```diff
@@ -0,0 +1,157 @@
+// This module is included in the following assemblies:
+// * create/using-manual-approval.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-behavior-of-approvaltask-with-groups-and-users_{context}"]
+= Behavior of ApprovalTask with groups and users
+
+The `ApprovalTask` custom resource (CR) introduces manual approval gates in Tekton pipelines. You can configure it with individual users, groups, or a combination of both.
+
+When an `ApprovalTask` is created, it automatically creates a group based on the specification defined in the pipeline. 
```

Comment:
The ApprovalTask custom resource (CR) provides a manual approval gate for Tekton pipelines. You can configure it to require approvals from individual users, groups, or a combination of both. When a PipelineRun reaches an approval step, the controller creates a corresponding ApprovalTask instance and pauses the pipeline until the required approvals are received.

Reply:
Fixed.